scripts/publish-packages.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
+
+TAG=${1:-dev}
+
+pnpm --filter @skybridge/devtools publish --tag "$TAG" --access public --ignore-scripts --no-git-checks
```

Comment:
You don't publish core ?

Reply:
Core is published in a dedicated script because it requires a specific npm_token (the one of your account) compared to the other packages which require the one I created for the skybridge org